game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
okay, but other thing is that this nettable you referenced is emitted only when game is loaded(after pregame hero pick screen), while event is done as soon as chosen